calligraphy/src/main/java/uk/co/chrisjenx/calligraphy/ReflectionUtils.java

```diff
@@ -50,8 +54,8 @@ static void invokeMethod(Object object, Method method, Object... args) {
         try {
             if (method == null) return;
             method.invoke(object, args);
-        } catch (IllegalAccessException | InvocationTargetException ignored) {
-            ignored.printStackTrace();
+        } catch (IllegalAccessException | InvocationTargetException e) {
+            Log.d(TAG, "Can't invoke method using reflection", e);
```

Comment:
I was trying to avoid `Log` calls as they don't get turned off properly. If you want to use this, thats fine but maybe add a static debug field in the CalligraphyConfig class.